OpenRA.Game/Sound/Sound.cs

```diff
@@ -348,9 +350,21 @@ public float VideoSeekPosition
 			var name = prefix + clip + suffix;
 
 			if (!string.IsNullOrEmpty(name) && (p == null || p == p.World.LocalPlayer))
-				soundEngine.Play2D(sounds[name],
+			{
+				ISound sound = soundEngine.Play2D(sounds[name],
```

Comment:
OpenRA code style is to use `var` instead of explicit types in unambiguous cases like these.